extensions/wikia/UserLogin/WikiaConfirmEmailSpecialController.class.php

```diff
@@ -109,7 +109,7 @@ public function index() {
 				return;
 			}
 
-			if ( $user->checkPassword( $this->password ) ) {
+			if ( $user->checkPassword( $this->password )->success() ) {
 				$this->wg->User = $user;
 
 				if ( $user->getGlobalFlag( UserLoginSpecialController::NOT_CONFIRMED_SIGNUP_OPTION_NAME ) != null ) {// Signup confirm
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)